.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: ci
+on: [push, pull_request]
```

Comment:
This essentially creates duplicate builds for PRs... it matches the current behavior with Travis, but since GHA displays each job as a separate check, it clutters the merge check status.

Should we disable CI on push? It seems most valuable for PRs.